mysql-test/suite/rocksdb/t/rocksdb_cf_options.test

```diff
@@ -16,14 +16,10 @@ insert into t1 values (1);
 insert into t2 values (2);
 insert into t3 values (2);
 
-# set cf_options for non-existent cf5, cf5 should not be created. Error should be reported.
-call mtr.add_suppression("Column family 'cf5' not found.");
---error ER_WRONG_VALUE_FOR_VAR 
+# set cf_options for non-existent cf5, cf5 should be created automatically
```

Comment:
@inikep this looks like a change in behavior.  In PS, we do not want automatic creation of missing column families, see https://www.percona.com/doc/percona-server/5.7/myrocks/variables.html#rocksdb_create_missing_column_families

Reply:
`ROCKSDB_CREATE_MISSING_COLUMN_FAMILIES` is `OFF` for both Percona and upstream. The above change reverts `PS-6125` because it's in conflict with `drop_cf` and the issue described at `PS-6125` was already fixed by upstream with https://github.com/facebook/mysql-5.6/commit/876d9a3fafe36d0cf6b285b1496f1de94eb89739

Reply:
I still don't understand why the change in behavior.  With rcmcf off, this should fail if cf5 does not exist.

Reply:
Changes were discarded